src/components/Footer/Footer.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Link } from 'react-router';
+import s from './Footer.module.css';
+
+const Footer = () => {
+  return (
+    <div>
```

Comment:
You don't need to wrap it into div, because the `footer` element is a block like the `div` does